harness.sh

```diff
@@ -143,6 +143,7 @@ postgresport=${POSTGRES_PORT}
 postgresuser=${POSTGRES_USER}
 postgrespass=${POSTGRES_PASS}
 postgresdbname=${POSTGRES_DBNAME}
+purgedb=1
```

Comment:
I dont think `purgedb` should be set permanently in the config file. My workflow often looks like this:

1. Start the harness
1. Test dcrpool features
1. Make code changes
1. Rebuild dcrpool
1. Restart dcrpool in tmux
1. Goto 2.

This workflow wont be possible if dcrpool purges every time it starts up.

Perhaps we should set `--purgedb` on the CLI instead, towards the bottom of this file, so it is only used the first time?